doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -0,0 +1,169 @@
+Upgrade to Flower Next
+======================
+
+Welcome to the migration guide for updating Flower to Flower Next! Whether you're a seasoned user
+or just getting started, this guide will help you smoothly transition your existing setup to take
+advantage of the latest features and improvements in Flower Next, starting from version 1.8. Let's dive in!
+
+Install update
+--------------
+
+Here's how to update an existing installation of Flower to Flower Next with ``pip``:
+
+.. code-block:: bash
+    
+    $ python -m pip install -U flwr
+
+or if you need Flower Next with simulation:
+
+.. code-block:: bash
+    
+    $ python -m pip install -U flwr[simulation]
+
+Dependencies
+------------
+Ensure you set the following version constraint in your ``requirements.txt``
+
+.. code-block:: 
+
+    flwr>=1.8.0,<2.0
+
+or ``pyproject.toml``:
+
+.. code-block:: toml
+
+    dependencies = ["flwr>=1.8.0,2.0"]
+
+
+Required changes
+----------------
+
+In Flower Next, the *infrastructure* and *application layers* have been decoupled.
+Therefore, the main changes are in the setup of the ``SuperNode``, ``ClientApp`,
+``ServerApp`` and execution of federated learning. These are the following non-breaking
+changes that require manual updates.
+
+``SuperNode``/``ClientApp``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Use ``flwr.client.ClientApp(client_fn)`` instead of ``flwr.client.start_client(server_address, client)``.
```

Comment:
```suggestion
- Wrap your existing client with ``flwr.client.ClientApp()`` instead of launching it via ``flwr.client.start_client()``.
```

Reply:
Should we point readers to the documentation of each of these classes?

Reply:
Good suggestion. I'll add them.